integration_tests/sdk/aqueduct_tests/checks_test.py

```diff
@@ -123,13 +123,6 @@ def foo(table):
     flow_run = flow.latest()
     assert flow_run.artifact("foo artifact").get()
 
-    # TODO(ENG-2629): Fetching a warning check should not raise an exception.
-    with pytest.raises(
-        ArtifactNeverComputedException,
-        match="This artifact was part of an existing flow run but was never computed successfully",
-    ):
-        assert not flow_run.artifact("foo artifact (1)").get()
```

Comment:
Wait we should definitely keep this assertion in, since it should work now right?

Reply:
Agree, get should fetch some value now, running the tests again.